OneSignalSDK/onesignal/in-app-messages/src/main/java/com/onesignal/inAppMessages/internal/display/impl/WebViewManager.kt

```diff
@@ -279,13 +279,14 @@ internal class WebViewManager(
             }
             Logging.debug("In app message, showing first one with height: $newHeight")
 
-            messageView!!.setWebView(webView!!)
+            messageView?.setWebView(webView!!)
             if (newHeight != null) {
                 lastPageHeight = newHeight
-                messageView!!.updateHeight(newHeight)
+                messageView?.updateHeight(newHeight)
             }
-            messageView!!.showView(activity)
-            messageView!!.checkIfShouldDismiss()
+            messageView?.showView(activity)
+            // Executed in the same thread
```

Comment:
I don't understand the importance of this comment. Isn't everything in this function running in the same thread?

Reply:
Updated comment for clarity!